contracts/test/foundry/FleekERC721/TokenURI.t.sol

```diff
@@ -57,7 +57,13 @@ contract Test_FleekERC721_TokenURI is Test_FleekERC721_Base, Test_FleekERC721_To
         CuT.setTokenExternalURL(tokenId, "https://new-url.com");
         transferENS("new-ens.eth", deployer);
         CuT.setTokenENS(tokenId, "new-ens.eth");
-        CuT.setTokenBuild(tokenId, "ce1a3fc141e29f8e1d00a654e156c4982d7711bf", "https://github.com/other/repo");
+        CuT.setTokenBuild(
+            tokenId,
+            "ce1a3fc141e29f8e1d00a654e156c4982d7711bf",
+            "https://github.com/other/repo",
+            "ipfsHash",
+            "domain"
```

Comment:
Maybe is a good idea to use mocked real values for the testing suits.

Reply:
it's going to be a similar string value like `mtwirsqawjuoloq2gvtyug2tc3jbf5htm2zeo4rsknfiv3fdp46a` (hello world)